### PR TITLE
Updating Translations 2005 4/29/2020

### PR DESCRIPTION
**Pull request by BrightspaceGitHubReader**

### CODE REVIEW

File: locales/sv-SE.json [hunk 1]
```diff
@@ -96,14 +99,18 @@
...
    "previousPage" : "Föregående sida",
+   "printExportRecords" : "Skriv ut/exportera poster",
    "printRecords" : "Skriv ut poster",
    "progress" : "Framsteg",
    "progressByType" : "Status efter typ",
    "progressCurrentPeriod" : "Status för aktuell period",
    "questionName" : "Frågetext",
    "questionNamePlaceholder" : "Ange frågetext",
    "range" : "{rangeStart}–{rangeEnd}",
+   "rangeOnlyEnd" : "Poster till och med {RangeEnd}",
```
Comment (jwong-d2l) [inline locales/sv-SE.json:110]: @BraydonKains or @TomMichielsD2L , are lang terms case sensitive?  I forget.

File: locales/pt-BR.json [hunk 2]
```diff
@@ -96,14 +99,18 @@
...
    "previousPage" : "Página Anterior",
+   "printExportRecords" : "Imprimir/Exportar Registros",
    "printRecords" : "Imprimir Registros",
    "progress" : "Progresso",
    "progressByType" : "Progresso por Tipo",
    "progressCurrentPeriod" : "Progresso no Período Atual",
    "questionName" : "Texto da Pergunta",
    "questionNamePlaceholder" : "Insira o texto da pergunta",
    "range" : "{rangeStart} – {RangeEnd}",
+   "rangeOnlyEnd" : "Grava até e incluindo (rangeEnd)",
```
Comment (jwong-d2l) [inline locales/pt-BR.json:110]: they killed our bracket!
```suggestion
   "rangeOnlyEnd" : "Grava até e incluindo {rangeEnd}",
```

File: locales/pt-BR.json [hunk 3]
```diff
@@ -96,14 +99,18 @@
...
+   "rangeOnlyStart" : "Grava em ou após (rangeStart)",
```
Comment (jwong-d2l) [inline locales/pt-BR.json:111]:
```suggestion
   "rangeOnlyStart" : "Grava em ou após {rangeStart}",
```

File: locales/sv-SE.json (hunk 1 above)
Comment (TomMichielsD2L) [reply locales/sv-SE.json:110]: I believe that they are.

File: locales/pt-BR.json [hunk 4]
```diff
@@ -96,14 +99,18 @@
...
    "percent" : "{percent}%",
    "previousPage" : "Página Anterior",
+   "printExportRecords" : "Imprimir/Exportar Registros",
    "printRecords" : "Imprimir Registros",
    "progress" : "Progresso",
    "progressByType" : "Progresso por Tipo",
    "progressCurrentPeriod" : "Progresso no Período Atual",
    "questionName" : "Texto da Pergunta",
    "questionNamePlaceholder" : "Insira o texto da pergunta",
    "range" : "{rangeStart} – {RangeEnd}",
```
Comment (jwong-d2l) [inline locales/pt-BR.json:109]:
```suggestion
   "range" : "{rangeStart} – {rangeEnd}",
```

File: locales/sv-SE.json (hunk 1 above)
Comment (jwong-d2l) [inline locales/sv-SE.json:110]:
```suggestion
   "rangeOnlyEnd" : "Poster till och med {rangeEnd}",
```

File: locales/sv-SE.json [hunk 5]
```diff
@@ -96,14 +99,18 @@
...
+   "rangeOnlyStart" : "Poster från och med {RangeStart}",
```
Comment (jwong-d2l) [inline locales/sv-SE.json:111]:
```suggestion
   "rangeOnlyStart" : "Poster från och med {rangeStart}",
```

**Pull request merged**